### ISSUE TITLE
Echo engineering team lines of effort 2020

### ISSUE THREAD

**Issue by hschallhorn** [department-of-veterans-affairs/caseflow#13175]
Starting a new ticket for the new year! This is a continuation of #7556.

**Comment by hschallhorn** [comment]
# [Transcontinental](https://www.sun-sentinel.com/news/fl-xpm-2001-12-24-0112230419-story.html) sprint (Jan 13 - 26) engineering priorities
## SCM: Acting on Behalf of Others #11801
1. :star: (ALEC) Add Judge & Attorney Selectors to JudgeTeam Team Page #12433 (completion blocked by #12422)
1. :star: ⛔️ (HUNTER) JudgeTeam Admins can view JudgeTeam Judge's Queues #12423
1. :star: ⛔️ Link Judge Actions to Admin, Crediting Judge #12424
## Inactive Users #12523
1. ✅ :star: (KEVIN) Do not show inactive users as options for task assignee #12949
1. ✅ :star: (KAT) Mark judge teams inactive when the judge becomes inactive #12942
1. :star: (YOOM) Remove inactive users from all organizations #12940
## Bat Team Needs
1. (KAT) Ensure task paper trails are being saved when AMA appeals are cancelled #12617
## Stretch Goals
### Grand Unified Queue #10032
1. (KEVIN) Build attorney queues from config #11699
### Accessibility
1. Mouseover tool tips not accessible to keyboard-only or other AT users – they should be #12141
### Tasks
1. Prevent creation of child tasks of TimedHoldTasks #12037
### Tech Debt
1. Track down and solve a flake of your choosing #10516
1. Improved BVA Dispatch Tests #10886
### Documentation
1. Extract common task creation paths [#1226](https://github.com/department-of-veterans-affairs/appeals-team/issues/1226)
1. Add parent-child stats to Task Tree Documentation [#1234](https://github.com/department-of-veterans-affairs/appeals-team/issues/1234)

**Comment by hschallhorn** [comment]
# [Coast Guard](https://en.wikipedia.org/wiki/United_States_Coast_Guard) sprint (Jan 27 - Feb 9) engineering priorities
@yoomlam, @ajspotts, @kevmo, @lomky, @jimruggiero , and @hschallhorn, here are the priorities for this sprint!
## Priorities
### SCM: Acting on Behalf of Others #11801
1. :star: (ALEC) Add Judge & Attorney Selectors to JudgeTeam Team Page #12433 (completion blocked by #12422)
### Queue
1. Investigate how to pull appellant POA #10431
1. Deep dive into limitations of acting judges in caseflow #13136
### Batteam Needs
1. TypeError thrown when validation for AssignWidget fails on submit #10632
1. Cannot read property ‘isLegacyAppeal’ of undefined in SelectRemandReasonsView #13216
## Stretch Goals
### Tango
1. Email addresses for virtual hearings are editable after hearing date #13215
### Tasks
1. Handle single failure in batch assignment of attorney tasks #13110
1. Fix RecordInvalid: Validation failed: Assigned to has to be an attorney #12982
### Queue
1. (KEVIN) Build attorney queues from config #11699
### Accessibility #12095
1. Filter button functionality not accessible using JAWS keyboard navigation #12902
1. In Document Viewer: Sidebar section headers (e.g. Categories, Issues) cannot be accessed via keyboard – users should be able to tab to these elements and operate them via keyboard #12145
### Tech Debt
1. Track down and solve a flake of your choosing #10516
1. (KAT) Improved BVA Dispatch Tests #10886
1. Clean both databases between every test #12725
### Documentation
1. Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
1. (YOOM) Extract common task creation paths [#1226](https://github.com/department-of-veterans-affairs/appeals-team/issues/1226)
1. Populate Task descriptions [#1235](https://github.com/department-of-veterans-affairs/appeals-team/issues/1235)

**Comment by hschallhorn** [comment]
# [Friendship 7](https://en.wikipedia.org/wiki/Mercury-Atlas_6) sprint (Feb 10 - 24) engineering priorities
@yoomlam, @ajspotts, @kevmo, @lomky, @jimruggiero, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Priorities
### Queue
1. ⭐️ Deep dive into limitations of acting judges in caseflow #13136
### Batteam Needs
1. TypeError thrown when validation for AssignWidget fails on submit #10632
### Data Cleanup
1. Fix tasks that were put on hold without a child TimedHoldTask when reassigned #12480
1. Move completed AttorneyTasks to new JudgeDecisionReviewTask on reassign #13279
1. Clean up PreRoutingFoiaColocatedTasks #13266
1. Investigate odd mail task trees #13269
## Stretch Goals
### Tasks
1. Handle single failure in batch assignment of attorney tasks #13110
1. (YOOM) Fix RecordInvalid: Validation failed: Assigned to has to be an attorney #12982
### Queue
1. Build judge queues from config #11700
1. (KEVIN) Build attorney queues from config #11699
### Accessibility #12095
1. In Document Viewer: Sidebar section headers (e.g. Categories, Issues) cannot be accessed via keyboard – users should be able to tab to these elements and operate them via keyboard #12145
### Tech Debt
1. fix FactoryBot's creation of parent tasks assigned to different appeal #13130
1. Track down and solve a flake of your choosing #10516
1. Clean both databases between every test #12725
### Documentation
1. Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
1. Include user roles information in Task Tree Documentation [#1225](https://github.com/department-of-veterans-affairs/appeals-team/issues/1225)

**Comment by hschallhorn** [comment]
# [Peace Corps](https://en.wikipedia.org/wiki/Peace_Corps) sprint (Feb 24 - Mar 8) engineering priorities
@yoomlam, @ajspotts, @kevmo, @lomky, @jimruggiero, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Priorities
### Special case movement #11801
1. :star: SCM: SCM Team can view Judge's Assign Queue #12423
1. :star: SCM: Allow SCM Team members to perform actions for judges #12424
### Queue
1. (KAT) :star: Deep dive into limitations of acting judges in caseflow #13136 
### Accessibility #12095
1. In Document Viewer: Sidebar section headers (e.g. Categories, Issues) cannot be accessed via keyboard – users should be able to tab to these elements and operate them via keyboard #12145
## Stretch Goals
### Data Cleanup
1. Investigate odd mail task trees #13269
### Tasks
1. (ALEC) Handle single failure in batch assignment of attorney tasks #13110
### Grand Unified Queue #10032
1. Build judge queues from config #11700
1. (KEVIN) Build attorney queues from config #11699
### Batteam Needs
1. Reword and Clarify 'Cancel Task' action on AttorneyTask #13362
1. Don't alert if RootTask is already on_hold or RootTask should be updated #13278
### Tech Debt
1. Track down and solve a flake of your choosing #10516
1. fix FactoryBot's creation of parent tasks assigned to different appeal #13130
1. Clean both databases between every test #12725
### Documentation
1. Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
1. Include user roles information in Task Tree Documentation [#1225](https://github.com/department-of-veterans-affairs/appeals-team/issues/1225)

**Comment by hschallhorn** [comment]
# [American Revolution](https://en.wikipedia.org/wiki/American_Revolutionary_War) sprint (Mar 9 - Mar 23) engineering priorities
@yoomlam, @ajspotts, @lomky, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Special case movement #11801
1. (ALEC) :star: SCM: Allow SCM Team members to perform actions for judges #12424
1. Judge Assign page should display the tasks of the user_id provided in the URL #13597
### Queue
1. (KAT) :star: Deep dive into limitations of acting judges in caseflow #13136
### Support needs
1. :star: Adding attorneys to judge team #13616
### Universal Case Title #8211
1. Add badges to Universal Case Title #8213
### Accessibility #12095
1. Table in Queue > Reader is not coded as a table. Filter and sort not accessible to AT #12136
## Bonus Set
### Tasks
1. Case Reassignment #13583
1. (HUNTER) Handle single failure in batch assignment of attorney tasks #13110
### Batteam Needs
1. Reword and Clarify 'Cancel Task' action on AttorneyTask #13362
1. Don't alert if RootTask is already on_hold or RootTask should be updated #13278
### Tech Debt
1. Track down and solve a flake of your choosing #10516
1. (YOOM) fix FactoryBot's creation of parent tasks assigned to different appeal #13130
1. Clean both databases between every test #12725
### Documentation
1. Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
1. Include user roles information in Task Tree Documentation [#1225](https://github.com/department-of-veterans-affairs/appeals-team/issues/1225)

**Comment by hschallhorn** [comment]
# [Sandra Day O’Connor](https://en.wikipedia.org/wiki/Sandra_Day_O%27Connor) sprint (Mar 23 - Apr 5) engineering priorities
@yoomlam, @ajspotts, @lomky, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Special case movement #11801
1. ⭐️ (ALEC/HUNTER) Allow SCM Team members to perform actions for judges #12424
1. Give judge assign queues human readable urls #13433
### Special Issues
1. :star: Legacy Appeals: Add special issues #13396
1. :star: Special Issues Page: Add Validation #13735
1. :star: Remove “(optional)” from special issues page for legacy appeals #9638
### AVLJ Legacy cases
1. Validate document_id when checking if a decision has been written for a case #13688
## Bonus Set
### Reader
1. :star: VBMS API Update to Support Correct Display of Labels in Reader #12979
### Cancelled appeals
1. Make cancelled appeals more obvious in appeals search #13654
1. Do not show VLJ or Attorney for cancelled appeal in case details #13747
### Tasks
1. (HUNTER) Handle single failure in batch assignment of attorney tasks #13110
### Batteam Needs
1. Reword and Clarify ‘Cancel Task’ action on AttorneyTask #13362
1. Don’t alert if RootTask is already on_hold or RootTask should be updated #13278
### Tech Debt
1. Track down and solve a flake of your choosing #10516 flaky test tickets
1. Clean both databases between every test #12725
### Documentation
1. Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
1. Include user roles information in Task Tree Documentation #1225
### Metrics
1. (YOOM) Identify impactful metrics to implement #13642

**Comment by hschallhorn** [comment]
# [Civil Rights](https://en.wikipedia.org/wiki/Civil_Rights_Act_of_1866) sprint (Apr 6 - Apr 19) engineering priorities
@yoomlam, @ajspotts, @lomky, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Special case movement #11801
1. :star: Allow SCM Team members to perform legacy actions for judges #12424
### Special Issues #13423
2. AMA: Add Special Issues Page #13736
### OT Cases #13418
1. Show Overtime Cases #13417
### Foxtrot Needs
1. Queue Org | Create new 'Case Review' team and copy permissions | Update Original 'Case Review' team name to 'Intake' team #13865
## Bonus Set
### Reader
1. (ALEC) :star: VBMS API Update to Support Correct Display of Labels in Reader #12979
### a11y #12095
1. Fix labels with a parenthetical pluralization #12217
### Cancelled appeals
1. (STEPHEN) Make cancelled appeals more obvious in appeals search #13654
1. (BLOCKED) Do not show VLJ or Attorney for cancelled appeal in case details #13747
### Tasks
1. Handle single failure in batch assignment of attorney tasks #13110
### Batteam Needs
1. AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
1. Track down and solve a flake of your choosing #10516 flaky test tickets
1. Clean both databases between every test #12725
### Documentation
1. Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
### Metrics
1. (YOOM) Debugging why reporting job is not running -- connects #13770

**Comment by hschallhorn** [comment]
# [Library of Congress](https://en.wikipedia.org/wiki/Civil_Rights_Act_of_1866) sprint (Apr 20 - May 3) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Special case movement #11801
- [ ] ⭐️ Make /queue/<css_id>/assign the default url to reach a judge’s assign page #13913
- [ ] Handle mixed case css_ids when navigating to a judge’s assign queue #13912
### Special Issues #13423
- [ ] ⭐️ BVA Dispatch: Add Validation to Special Issues #14018
### OT Cases #13418
- [ ] ⭐️ Mark Cases as OT #13415
### Board Needs
- [ ] ⭐️ Route EvidenceOrArgumentMailTasks to the mail team upon creation #13671
- [ ] Sync up permissions for creating a team and modifying a team #13748
- [ ] Request to realign the Virginia group from VSO to FieldVSO [#114](https://github.com/department-of-veterans-affairs/dsva-vacols/issues/114)
### Sentry Alerts
- [ ] EstablishClaim returns a TypeError for null error object #13405
## Bonus Set
### a11y #12095
- [ ] Enable list Axe rule for Caseflow #12885
### Cancelled appeals
- [ ] Clarify language when the last issue is removed on a case #13925
### Tasks
- [ ] Handle single failure in batch assignment of attorney tasks #13110
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean both databases between every test #12725
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692

**Comment by yoomlam** [comment]
# [David Hunter](https://en.wikipedia.org/wiki/David_Hunter) sprint (May 4 - May 18) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Inactive users #12523
- [ ] Caseflow user status management allow search by name [#14089](https://github.com/department-of-veterans-affairs/caseflow/issues/14089)
### Special case movement #11801
- [ ] :star: Handle bad css_id when navigating to a judge assign queue [#13911](https://github.com/department-of-veterans-affairs/caseflow/issues/13911)
### OT Cases #13418
- [ ] (Yoom & Alec) :star: Mark Cases as OT [#13415](https://github.com/department-of-veterans-affairs/caseflow/issues/13415)
### Board Needs
- [ ] :star: Route EvidenceOrArgumentMailTasks to the mail team upon creation [#13671](https://github.com/department-of-veterans-affairs/caseflow/issues/13671)
- [ ] VLJ Support: Task auto-distribution discrepancy [#14084](https://github.com/department-of-veterans-affairs/caseflow/issues/14084)
### Batteam Needs  
- [ ] Defer the loading of user 'on-hold' tasks [#14142](https://github.com/department-of-veterans-affairs/caseflow/issues/14142)
### Sentry Alerts
- [ ] Handle undefined attorney in app/queue/AssignedCasesPage.jsx [#12999](https://github.com/department-of-veterans-affairs/caseflow/issues/12999)
## Bonus Set
### Tasks
- [ ] Do not use assigned_to_location to sort and filter by assignee in org queues [#13914](https://github.com/department-of-veterans-affairs/caseflow/issues/13914)
- [ ] Mark cases in_progress when a user clicks into a case details [#14023](https://github.com/department-of-veterans-affairs/caseflow/issues/14023)
### Sentry Alerts
- [ ] AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### a11y #12095
- [ ] Enable aria-required-children Axe rule for Caseflow #12883
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692
### Metrics
- [ ] (Yoom) Get stats on how documents are viewed and used in Reader #14040
- [ ] Get stats on user’s time spent reviewing documents #14044

**Comment by hschallhorn** [comment]
# [Amelia Earhart](https://en.wikipedia.org/wiki/Amelia_Earhart) sprint (May 4 - May 18) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Case movement #11801
- [ ] ⭐Case Movement_Non-Blocking Tasks | Assigned To Field Not Displaying Correct User #14214
- [ ] ⭐Update Copy in CM Actions Drop down #14182
- [ ] ⭐Case Movement_Non-Blocking Tasks | Make assign/reassign modals consistent #14210
- [ ] Case Movement_Non-Blocking Tasks | Screenreader does not interact with actions modal #14212
### OT Cases #13418
- [ ] Remove attorney functionality #13584
### Adding New Contentions #13423
- [ ] ⭐Validate that the special issues are not selected along with the "No Special Issues" #13995
- [ ] Validate against special issue selections made on Review Decision page #14269
### Board Needs
- [ ] ⭐Gather metrics on how many documents show up in reader for a given user #14289
### a11y #12095
- [ ] (KAT) Enable aria-required-children Axe rule for Caseflow #12883
## Bonus Set
### Batteam Needs
- [ ] (Hunter) Defer the loading of user ‘on-hold’ tasks #14142
### Grand Unified Queue #10032
- [ ] Build judge queues from config #11700
### Sentry Alerts
- [ ] AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692
### Metrics
- [ ] (Yoom/Peter) VHA ODS tables #13733
- [ ] Get stats on user’s time spent reviewing documents #14044

**Comment by hschallhorn** [comment]
# []() sprint (Jun 1 - Jun 14) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!

## Core Set
### Case movement #11801 - 70% Complete 
- [ ] ⭐  Case Movement_Non-Blocking Tasks | Assign Task Modal Selections Persist #14213
- [ ] ⭐  Case Movement_Non-Blocking Tasks | Display error when assigning/reassigning a task is unsuccessful #14216
- [ ] ⭐  Assigning a legacy case to attorney not successful in demo #14195
- [ ] ⭐  Case Movement_Non-Blocking Tasks | Assign/Reassign Case to Attorney Banner Inconsistency #14217

### OT Cases #13418 - 63% Complete
- [ ] ⭐  Remove OT status for cases that are reassigned #14366

### Board Needs
- [ ] ⭐  Investigate what occurs when Reader is refreshed #14298

### Adding New Contentions #13423 - 55% Complete
- [ ] Dispatch & Special Issues Investigation #14433
- [ ] "Back" Button State Issue #14273
- [ ] Queue | Screenreader does not read success messages #14272

### a11y #12095
- [ ] Queue: When a column has a filter applied, AT does not read to users that a filter is applied #12134

### Bulk Assignment
- [ ] Queue: Bulk Assignment Functionality All Organizations #14124

## Bonus Set
### Batteam Needs
- [ ]  (HUNTER) Defer the loading of user ‘on-hold’ tasks #14142
### Sentry Alerts
- [ ] AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692
### Metrics
- [ ] Get stats on user’s time spent reviewing documents #14044

**Comment by yoomlam** [comment]
# Stonewall sprint (Jun 15 - Jun 28) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Case movement #11801 - 86% Complete
- [ ] ⭐ Remove "Special" copy from Completed Task tab #14447
### Adding New Contentions #13423 - 58% Complete (Rolling Over)
- [ ] ⭐ Possibly replace w/Kat's investigation ticket
### a11y #12095
- [ ] ⭐ Queue: When a column has a filter applied, AT does not read to users that a filter is applied #12134 (Rollover possibly)
- [ ] ⭐ (YOOM) Reader Document Viewer: Page loses visual indication of focus after tabbing through `Search cases` #12245
### Case movement | Expediting cases to a judge for decision #11397 - 55% Complete
- [ ] Write a tech spec to allow tasks to block dispatch #14055
### Bulk Assignment

- [ ] (HUNTER) Queue: Bulk Assignment Functionality All Organizations [#14124](https://github.com/department-of-veterans-affairs/caseflow/issues/14124)

#### Board Needs
- [ ] Canceled/Deleted Appeals Still Showing In User Queues #14219
## Bonus Set
### a11y #12095
- [ ]  Table in Queue > Reader headers when sorted are not announced by screen reader #12136
### Batteam Needs
- [ ] (HUNTER) Defer the loading of user ‘on-hold’ tasks #14142
### Sentry Alerts
- [ ] AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692
### Metrics
- [ ] Get stats on time spent on tasks during Case Decision Review #14041

**Comment by hschallhorn** [comment]
# [Stonewall](https://en.wikipedia.org/wiki/Stonewall_riots) sprint (Jun 15 - Jun 28) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Case movement #11801 - 86% Complete
- [ ] ⭐ Remove "Special" copy from Completed Task tab #14447
### Adding New Contentions #13423 - 58% Complete (Rolling Over)
- [ ] ⭐ Queue Special Issues Questions #14524
### a11y #12095
- [ ] ⭐ Queue: When a column has a filter applied, AT does not read to users that a filter is applied #12134 (Rollover possibly)
- [ ] ⭐ Reader Document Viewer: Page loses visual indication of focus after tabbing through `Search cases` #12245
### Case movement | Expediting cases to a judge for decision #11397 - 55% Complete
- [ ] Write a tech spec to allow tasks to block dispatch #14055
### Bulk Assignment

- [ ] (HUNTER) Queue: Bulk Assignment Functionality All Organizations [#14124](https://github.com/department-of-veterans-affairs/caseflow/issues/14124)

#### Board Needs
- [ ] Canceled/Deleted Appeals Still Showing In User Queues #14219
## Bonus Set
### a11y #12095
- [ ]  Table in Queue > Reader headers when sorted are not announced by screen reader #12136
### Batteam Needs
- [ ] (HUNTER) Defer the loading of user ‘on-hold’ tasks #14142
### Sentry Alerts
- [ ] AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692
### Metrics
- [ ] Get stats on time spent on tasks during Case Decision Review #14041

**Comment by hschallhorn** [comment]
# [Thurgood Marshall](https://en.wikipedia.org/wiki/Thurgood_Marshall) sprint (Jun 29 - Jul 12) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### New Special Issues #13423
- [ ] ⭐ (Kat) Validate against special issue selections made on Review Decision Page #14269
### a11y #12095
- [ ] ⭐ (Alec) Queue: When a column has a filter applied, AT does not read to users that a filter is applied #12134 (Awaiting Confirmation from Lindsey to remove) 
- [ ] ⭐ Table in Queue > Reader headers when sorted are not announced by screen reader #12136
### Case Movement - Blocking Tasks | Expediting cases to a judge for decision #11397
- [ ] ⭐ (Alec) Write a tech spec to allow tasks to block dispatch #14055 
- [ ] Add reason field to cancel task modal #14351
### Automatic Case Distribution Algorithm Updates #14538
- [ ] ⭐ Create a 30 case backlog for legacy hearing cases associated with a judge #14543
- [ ] Investigate priority legacy cases that have been stuck at ready to distribute #14597
### Board Needs
- [ ] Cancelled Appeals not in Search #14221
### Bulk Assignment
-    [ ]  (HUNTER) Queue: Bulk Assignment Functionality All Organizations [#14124](https://github.com/department-of-veterans-affairs/caseflow/issues/14124)
## Bonus Set
### Batteam Needs
- [ ] (HUNTER) Defer the loading of user 'on-hold' tasks #14142
### Sentry Alerts
- [ ] AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Document All Existing FeatureFlags #13692
- [ ] Document Special Issues Findings #14609
### Metrics
- [ ] Get stats on time spent on tasks during Case Decision Review #14041

**Comment by hschallhorn** [comment]
# [Sea of Tranquility](https://en.wikipedia.org/wiki/New_Horizons) sprint (Jul 13 - Jul 26) engineering priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Automatic Case Distribution Algorithm Updates #14538
- [ ] ⭐ (HUNTER + KAT) Create Auto-Distribution Job for Priority Cases #14604
### Marking Cases as OT
- [ ] ⭐ Mark Cases as OT | Success Banner Remains Persistent #14667
- [ ] ⭐ Remove OT status for cases that are reassigned #14366
### Board Needs
- [ ] ⭐ Record AOD for claimants 75 & Older in Caseflow Database #14637
- [ ] (Kat) Cancelled Appeals not in Search #14221 (probably not rolling over)
- [ ] Collect information on RoundRobin Automatic Task Assignments #14629
## Bonus Set
### Documentation
- [ ] Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ] Create Dispatch Establish Claims controller schema to catch mismatch Special Issues #14659
- [ ] Document All Existing FeatureFlags #13692
- [ ] (KAT) Document Special Issues Findings #14609
### JudgeTeam Architecture Cleanup #13991
- [ ] Codify JudgeTeam one admin limit #13993
- [ ] Remove unneeded buttons from JudgeTeam Team Management #13992
### Queue
- [ ] Wrong cases in judge queue after assigning case to an attorney #14655
### Sentry Alerts
- [ ] (Alec) AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing #10516
- [ ] Clean up open tasks with closed parents #13438
### Metrics
- [ ] (Yoom) Get stats on time spent on tasks during Case Decision Review #14041

**Comment by hschallhorn** [comment]
# [Voting Rights](https://en.wikipedia.org/wiki/Voting_Rights_Act_of_1965) sprint (Jul 27 - Aug 9) engineering priorities
_See also [Shelby County v. Holder](https://en.wikipedia.org/wiki/Shelby_County_v._Holder#Impact)_

@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the priorities for this sprint!
## Core Set
### Automatic Case Distribution Algorithm Updates #14538 - 17% Complete
 - [ ] ⭐ (HUNTER) Create Auto-Distribution Job for Priority Cases #14604
 - [ ] ⭐ Update Case Orders Logic #14542
 - [ ] ⭐ Create DVC Teams #14462

### Case Movement  | Expediting cases to a judge for decision #11397 - 60% Complete
 - [ ] ⭐ Implement Backend Logic For Cancellation of Blocking Tasks #14260
 - [ ] ⭐ (ALEC) [Tech Spec] Allow tasks to block Dispatch #14661
### Board Needs
 - [ ] (KAT) Collect information on RoundRobin Automatic Task Assignments #14629
 - [ ] ⭐ (Yoom) Record AOD for claimants 75 & Older in Caseflow Database #14637

## Bonus Set
### Documentation
- [ ]  Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ]  Create Dispatch Establish Claims controller schema to catch mismatch Special Issues #14659
- [ ]  Document All Existing FeatureFlags #13692
- [ ] (KAT) Document Special Issues Findings #14609
- [ ] (KAT) Research Code Organization Patterns in Hearings and eFolder #14561
### JudgeTeam Architecture Cleanup #13991
- [ ] Codify JudgeTeam one admin limit #13993
- [ ] Remove unneeded buttons from JudgeTeam Team Management #13992
### Queue
- [ ] Wrong cases in judge queue after assigning case to an attorney #14655
### Tasks
- [ ] Store cancelled\_by\_id in task table #14365
### Sentry Alerts
- [ ] (Alec) AttorneyJudgeCheckoutError: Diagnostic code is not included in the list #10948
### Tech Debt
- [ ] Track down and solve a flake of your choosing
- [ ] Clean up open tasks with closed parents #13438
### Metrics
- [ ] (Yoom) Get stats on time spent on tasks during Case Decision Review #14041

**Comment by hschallhorn** [comment]
# [Social Security](https://en.wikipedia.org/wiki/Social_Security_Act) sprint (Aug 10 - Aug 23) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the engineering and design priorities for this sprint!
## Engineering Core Set
### Automatic Case Distribution Algorithm Updates #14538
 - [ ] ⭐ (HUNTER) Create Auto-Distribution Job for Priority Cases #14604
 - [ ] ⭐ (ALEC) Create DVC Teams #14462
 - [ ] ⭐ :no_entry_sign: (HUNTER) Create Front End for adding or removing VLJs from Auto-Distribution of Cases #14608 
 - [ ]  (YOOM - Weekly) Investigate priority legacy cases that have been stuck at ready to distribute #14597
### Case Movement  | Expediting cases to a judge for decision #11397
- [ ] ⭐ Allow tasks to block dispatch #14056
- [ ] ⭐ (KAT) Implement Backend Logic For Cancellation of Blocking Tasks #14260
- [ ] ⭐ :no_entry_sign: Allow CM Team to Expedite Cases with Blocking Tasks (Front end) #13780
## Engineering Bonus Set
### One queue to rule them all
- [ ] ⭐ Add tabs to judges' individual queue table views #10044
### Documentation
- [ ]  Pick a Backend Workgroup Tech Improvements Documentation ticket to work on #12778
- [ ]  Create Dispatch Establish Claims controller schema to catch mismatch Special Issues #14659
- [ ]  Document All Existing FeatureFlags #13692
- [ ] (KAT) Document Special Issues Findings #14609
- [ ] (KAT) Research Code Organization Patterns in Hearings and eFolder #14561
### JudgeTeam Architecture Cleanup #13991
- [ ] Codify JudgeTeam one admin limit #13993
### Queue
- [ ] Wrong cases in judge queue after assigning case to an attorney #14655
### Tasks
- [ ] Store cancelled\_by\_id in task table #14365
### Tech Debt
- [ ] Track down and solve a flake of your choosing
- [ ] Clean up open tasks with closed parents #13438
### Metrics
- [ ] (Yoom) Get stats on time spent on tasks during Case Decision Review #14041

---

## Design Core Set
### Support AMA CAVC Remands #10991
- [ ] CAVC Remand Design and Research: Phase 4 (death dismissals) [#14858](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/issues/department-of-veterans-affairs/caseflow/14858)
- [ ] CAVC Remand Design and Research: Phase 5 (90-day letter) [#14859](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/issues/department-of-veterans-affairs/caseflow/14859)
- [ ] CAVC Remand Design and Research: Phase 6 (retrieve 90-day letter response) [#14864](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/issues/department-of-veterans-affairs/caseflow/14864)
### Trigger receipt of completed IHPs #13700
- [ ] Design and research for trigger receipt of completed IHPs [#13800](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/issues/department-of-veterans-affairs/caseflow/13800)
## Design Bonus Set
- [ ] Bulk Assign | SSC reassign to Judge [#14786](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/issues/department-of-veterans-affairs/caseflow/14796)
- [ ] Add ability to pre-fill congresspeople to Congressional Inquiry tasks [#10696](https://app.zenhub.com/workspaces/caseflow-5915dd178f67e20b5553ba0c/issues/department-of-veterans-affairs/caseflow/10696)

**Comment by hschallhorn** [comment]
# [Edith S. Sampson](https://en.wikipedia.org/wiki/Edith_S._Sampson) sprint (Aug 24 - Aug 23) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the engineering and design priorities for this sprint!
## Engineering Core Set
### Automatic Case Distribution Algorithm Updates #14538
 - [ ]  (HUNTER) Create Auto-Distribution Job for Priority Cases #14604
 - [ ]  (ALEC) Create DVC Teams #14462
 - [ ] :no_entry_sign: (HUNTER) Create Front End for adding or removing VLJs from Auto-Distribution of Cases #14608 
 - [ ]  (YOOM - Weekly) Investigate priority legacy cases that have been stuck at ready to distribute #14597
### Case Movement  | Expediting cases to a judge for decision #11397
- [ ] (KAT) Allow tasks to block dispatch #14056
- [ ] Update tasks to block dispatch #14057 [3]
- [ ] (HUNTER) Allow CM Team to Expedite Cases with Blocking Tasks (Front end) #13780
### AOD Requests
- [ ] AOD Docket for Subsequent Appeals #14085 [3]
- [ ] Bulk Assign should prioritize AOD cases 14781 [2]
### Tech Debt
- [ ] Rollback new contention additions from Dispatch #14714 [2]
## Engineering Bonus Set
### Tech Debt
- [ ] Investigation: Discover any AMA appeal states that won't move forward automatically #14307 [3]
- [ ] Automatically pull descriptions for tasks from top of the file comments [#1261](https://github.com/department-of-veterans-affairs/caseflow/issues/10044) [3]
- [ ] Store cancelled_by_id in task table #14365 [3]
- [ ] Automate fix for RedistributedCase::CannotRedistribute Sentry alert # 14705 [3]
### Batteam
- [ ] Remove "Mark task complete" as an available action for BVADispatchTasks #14869 [1]



---

## Design Core Set
### New feature
- [ ] CAVC Remand Design and Research: Phase 6 (retrieve 90-day letter response) #14864
- [ ] CAVC Remand Design and Research: Phase 7 #14865
- [ ] IHP Add new task action to VLJ Support Organization #14977
- [ ] IHP VLJ Support: New task action for return to POA	#14979
## Design Bonus Set
### UI change
- [ ] Universal case title wrapping #8968
- [ ] Case timeline recommendations	#9382

**Comment by hschallhorn** [comment]
# [TBD](https://en.wikipedia.org/wiki/TBD) sprint (Sept 7 - Sept 20) priorities
@yoomlam, @ajspotts, @lomky, @mkerns355, @geronimoramos, @RyanRoyUX, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the engineering and design priorities for this sprint!
## Engineering Core Set
### Trigger Receipt of IHPs #13700
- [ ] Capture Stats on Legacy IHP Tasks #15003 [2]
- [ ] Capture Stats on AMA IHP Tasks #14997 [2]
### AOD Requests
- [ ] (HUNTER) AOD Docket for Subsequent Appeals #14085 [3]
### Board Needs
- [ ] Assigned by Field for JudgeDecisionReview Task on AMA Cases Displaying as Assigned VLJ #14842 [2]
### Tech Debt
- [ ] ⭐ (ALEC) Automate fix for RedistributedCase::CannotRedistribute Sentry alert #14705 [3]
- [ ] (Yoom) Investigation: Discover any AMA appeal states that won't move forward automatically #14307 [3]
- [ ] (HUNTER) Store cancelled_by_id in task table #14365 [3]
## Engineering Bonus Set
### Case Movement  | Expediting cases to a judge for decision #11397
- [ ] ⛔ (KAT) Allow tasks to block dispatch #14056 [3]
- [ ] ⛔ (KAT) Update tasks to block dispatch #14057 [3]
### Tech Debt
- [ ] Automatically pull descriptions for tasks from top of the file comments [#1261](https://github.com/department-of-veterans-affairs/appeals-team/issues/1261) [3]
### Batteam
- [ ] Remove "Mark task complete" as an available action for BVADispatchTasks #14869 [1]

---

## Design Core Set
### New feature
- [ ] CAVC Remand Design and Research: Phase 6 (retrieve 90-day letter response) #14864
- [ ] CAVC Remand Design and Research: Phase 7 #14865
## Design Bonus Set
### UI change
- [ ] Universal case title wrapping #8968
- [ ] Case timeline recommendations	#9382

**Comment by araposo-tistatech** [comment]
# XXX sprint (Sept 21 - Oct 02) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the engineering and design priorities for this sprint!

## Engineering Core Set
### Trigger Receipt of IHPs #13700
- [ ] Investigate why appeals with VSOs that write IHPs do not have IHP tasks #14889 [3]
### Route Request for IHP tasks to the appropriate VSO #12674
- [ ] Enabler for #14980: Create model to store IHP draft V-Drive path 15206 [3]
- [ ] Story: Automatically Create & Send Child IHP Task to IHP writing POA #14940 [2] 
### Board Needs
- [ ] Story: Assigned by Field for JudgeDecisionReview Task on AMA Cases Displaying as Assigned VLJ #14842 [2]
### Tech Debt
- [ ] ⭐ Reader documents list sort and search are cleared when hitting the back button #15173 [3]
## Engineering Bonus Set
### Case Movement | Expediting cases to a judge for decision #11397
- [ ] ⛔ Allow tasks to block dispatch #14056 P/O #11397 [3]
- [ ] ⛔ Update tasks to block dispatch #14057 P/O #11397 [3]
### Tech Debt
- [ ] Automatically pull descriptions for tasks from top of the file comments #1261 [3]
- [ ] Discover and correct improperly undispatched appeals #14884 [3]
- [ ] Fix missing Legacy cases after case is signed #13872 [3]
## Design Core Set
### New feature
- [ ] CAVC Remand Design and Research: Phase 7 (judge checkout flows) - Extensions #14865
- [ ] CAVC Remand Design and Research: Phase 6 (retrieve 90-day letter response) – Mail Team #14864
## Design Bonus Set
### Metrics/Data
- [ ] Product Operations Dashboard for Team Echo

**Comment by lomky** [comment]
# [Eleanor Roosevelt](https://en.wikipedia.org/wiki/Eleanor_Roosevelt) sprint (Oct 5 - Oct 16) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @mbaenatan, @araposo-tistatech, and @hschallhorn, here are the engineering and design priorities for this sprint!
## Engineering Core Set
### ACD Algorithim Updates
 - [ ] [3] Fix priority logic for appeals with aod motions for appeal attribute caching #15353
 - [ ] [2] Fix priority logic for appeals with aod motions for distributions #15352
 - [ ] [3] Remove VLJ assignment requirement from AMA Hearing Cases #15343

### Board Needs
- [ ] (ALEC)  [2] Assigned by Field for JudgeDecisionReview Task on AMA Cases Displaying as Assigned VLJ #14842

## Engineering Bonus Set
### VSO Changes
 - [ ] :star: [2] Update VSOs to FieldVSOs #15310

### CAVC Remands
 - [ ] :star: [2] Spike: Investigate Create New Appeal Stream #15366

### Tech Debt
 - [ ] [2] Research Code Organization Patterns in Queue #14558

---

## Design Core Set
 - [ ] CAVC Remand Design and Research: Phase 7 (judge checkout flows) - Extensions #14865
 - [ ] Granted substitutions #13131

## Design Bonus Set
 - [ ] Product operations dashboard for team Echo

**Comment by lomky** [comment]
# [Dizzy Gillespie](https://en.wikipedia.org/wiki/Dizzy_Gillespie) sprint (Oct 19 - Oct 30) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @mbaenatan, @araposo-tistatech, @hschallhorn, @aridge1, and @drsjt1  here are the engineering and design priorities for this sprint!

## Engineering Core Set
### CAVC Remands #15219
 - [ ] [3] Create New Appeal Stream #14792
 - [ ] [2] New Organization: CAVC Litigation Support Team #15278
 - [ ] [3] CAVC Remand: Case Details Information Section #15227

### Tech Debt
 - [ ] [3] Recover from SSL_connect error in UpdateCachedAppealsAttributesJob #15416
 
## Engineering Bonus Set
### Tech Debt
 - [ ] [2] Cancellation of an AttorneyRewriteTasks should reactivate JudgeDecisionReviewTask, not JudgeAssignTask #15218
 - [ ] [2] Add validation checks where we expect only one open task #15220
 - [ ] [2] Create job to check for users that cannot be found in VACOLS #15268
 - [ ] [2] Add parent task validation to task classes #15349
 - [ ] [2] Investigate why appeals with VSOs that write IHPs do not have IHP tasks #14889
 - [ ] [2] Limit the number of AOD motions per appeal #15280
 - [ ] [2] Research Code Organization Patterns in Queue #14558

---

## Design Core Set
 - [ ] Creating research plan for granted substitutions #13131

**Comment by hschallhorn** [comment]
# [Veterans Day](https://www.va.gov/opa/vetsday/vetdayhistory.asp) sprint (Nov 2 - 15) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @mbaenatan, @araposo-tistatech, @hschallhorn, @aridge1, and @drsjt1  here are the engineering and design priorities for this sprint!

## Engineering Core Set
### CAVC Remands #15219
 - [ ] [3] (Alec) CAVC Remand: Case Details Information Section #15227
- [ ] [3] CAVC Remand: Litigation Support Form (Front End) #14795
- [ ] [3] Litigation Support: Create new `SendCavcRemandProcessedLetterTask` type #15076
- [ ] [2] Confirm default ordering of tasks prioritizes ama cavc remands #15308

### IHP Notifications #13700
- [ ] [5] Add Hover Information to Attorney & Judge Queue Days Waiting Column #15405
  - [ ] (HUNTER) Backend
  - [ ] Frontend

### Tech Debt
- [ ] [2] Create job to check that `on_hold` tasks should have an `active` descendant task #15267
 
## Engineering Bonus Set
### Tech Debt
 - [ ] [2] Add validation checks where we expect only one open task #15220
 - [ ] [2] (HUNTER) Add parent task validation to task classes #15349
 - [ ] [2] Investigate why appeals with VSOs that write IHPs do not have IHP tasks #14889
 - [ ] [2] Limit the number of AOD motions per appeal #15280
- [ ] [3] Handle EvidenceSubmissionWindowTasks with last_submitted_at edge cases #15245

---

## Design Core Set
 - [ ] Creating research plan for granted substitutions #13131

**Comment by lomky** [comment]
# [Spotswood](https://en.wikipedia.org/wiki/Alexander_Spotswood) sprint (Nov 16 - 27) priorities
@yoomlam, @ajspotts, @lomky, @geronimoramos, @mbaenatan, @araposo-tistatech, @hschallhorn here are the engineering and design priorities for this sprint!

## Engineering Core Set
[Migrating to tracking Core work in JIRA](https://vajira.max.gov/secure/RapidBoard.jspa?rapidView=1393&projectKey=CASEFLOW)

### CAVC Remands #15219
- [ ] [3] (Alec) CAVC Remand: Litigation Support Form (Front End) #14795
- [ ] [5] Add Admin Actions to 'SendCavcRemandProcessedLetterTask' [CASEFLOW-107](https://vajira.max.gov/browse/CASEFLOW-107)
- [ ] [2] Create a CavcRemandProcessedLetterResponseWindowTask upon completion of a SendCavcRemandProcessedLetterTask [CASEFLOW-108](https://vajira.max.gov/browse/CASEFLOW-108)
- [ ] [2] Allow CAVC Litigation Support Users to Add Issues on newly created appeal [CASEFLOW-106](https://vajira.max.gov/browse/CASEFLOW-106)
- [ ] [3] CAVC Remand: Update Auto-Distribution Criteria for Priority Cases [CASEFLOW-110](https://vajira.max.gov/browse/CASEFLOW-110)
- [ ] [2] Allow appeals to be searched by their stream docket number [CASEFLOW-343](https://vajira.max.gov/browse/CASEFLOW-343)

### Tech Debt
 - [ ] [3] Investigate why appeals with VSOs that write IHPs do not have IHP tasks [CASEFLOW-342](https://vajira.max.gov/browse/CASEFLOW-342)
 
## Engineering Bonus Set
### Tech Debt
 - [ ] [2] Add validation checks where we expect only one open task #15220
 - [ ] [2] Investigate why appeals with VSOs that write IHPs do not have IHP tasks #14889
 - [ ] [2] (Matt) Limit the number of AOD motions per appeal #15280
 - [ ] [3] (HUNTER) Handle EvidenceSubmissionWindowTasks with last_submitted_at edge cases #15245
 - [ ] [3] Do not use multiple task serializes for paginated tasks #14291
---

## Design Core Set
TK

**Comment by lomky** [comment]
Retiring this issue as of PI6!

[Committed Sprint Work (Core) is now tracked in JIRA](https://vajira.max.gov/secure/RapidBoard.jspa?rapidView=1393&projectKey=CASEFLOW)
[Uncommited Work (Bonus) is now tracked here](https://dsva.slack.com/archives/CQCUMFA0H/p1607965460006300)
